### ISSUE TITLE
Comment here if you want an OSD IRC chat account

### ISSUE THREAD

**Issue by bnvk** [opensourcedesign/organization#70]
As per #69 and making it easier to participate in OSD discussions- our old school IRC room #opensourcedesign has been a bit neglected- we should change this 😄 I've setup a [Lounge](https://thelounge.github.io) instance on my server specifically for OSD members to use. This will be accessible at [chat.opensourcedesign.net](http://chat.opensourcedesign.net) once @jancborchardt changes the domain record to point there.

I will need to make you an account on this chat server, so leave a comment here or email me (preferred) at [hi@brennannovak.com](mailto:hi@brennannovak.com) if you want one with the following info:

- Name:
- IRC nickname (if you have):

Once login you can change your password and change your settings and such!

**Comment by jancborchardt** [comment]
@bnvk huge thanks for setting this up! :)

One thing only – could we use the default theme? Then we can check out the default Lounge experience best and also see where they need design help. :)

**Comment by bnvk** [comment]
> One thing only – could we use the default theme?

I can switch the theme to default, but it's basically the same thing with just 
different colors- I picked this as it's closer to our color palate (b&w)

> Then we can check out the default Lounge experience best and also see
> where they need design help. :)

I think a lot of what we were doing in
[Slick](https://github.com/opensourcedesign/slick) are still ahead in terms
of user friendlyness than these themes.

**Comment by jancborchardt** [comment]
@bnvk ok, but the problem is that it was themes, which barely anyone changes. That’s why we actually need to get design pull requests into the Lounge core to improve it.

**Comment by grahamperrin** [comment]
At https://github.com/opensourcedesign/opensourcedesign.github.io/issues/56#issuecomment-285972020 @bnvk wrote: 

> … chat.opensourcedesign.net which i'm committed to keeping running.

– https://chat.opensourcedesign.net/ offers no hint of how to register so I manually reduced the URL to http://opensourcedesign.net/ then followed the invitation to "Join our Group on GitHub." with reference to https://github.com/opensourcedesign/organization/issues/29, which I imagined would be prerequisite to use https://chat.opensourcedesign.net/ 

After I received confirmation of membership without a note of how to chat, I sought other issues in GitHub and found this one `#70`. So, I am: 

- Graham Perrin
- IRC nickname grahamperrin

**Comment by simonv3** [comment]
@bnvk is there a reason this isn't open to anyone who wants to join?

**Comment by bnvk** [comment]
> @bnvk is there a reason this isn't open to anyone who wants to join?

Technological limitations of Lounge / IRC. Preventing spam / my server
getting k-lined.

**Comment by simonv3** [comment]
The second aspect could probably be prevented with a captcha? Shame that the Lounge doesn't allow general populace signing up :(

**Comment by bnvk** [comment]
Yes. Now mind you, this is for "a user account" on a lounge instance, not
generic signin to IRC.

My thinking to solve this has to auto-generate lounge accounts based on people
joining the github org, This should be achievable with a lil bit of bash
scripting and cron jobs.

**Comment by itsN1X** [comment]
Nikhil
## itsN1X

**Comment by grahamperrin** [comment]
@bnvk ping, please can you (or anyone) let me have credentials? I _would_ like to see what's in the lounge.

**Comment by simonv3** [comment]
@grahamperrin It's just IRC, so if you're used to using IRC you can join us at #opensourcedesign on freenode. 

This is exactly why I don't think this is a sustainable way of doing it.

**Comment by grahamperrin** [comment]
Thanks @bnvk for credentials, thanks @simonv3 for putting things in context. I joined ircs://chat.freenode.net/#opensourcedesign ten days ago, somehow I didn't realise that this issue `#70` was for a web interface to IRC. 

## An alternative to _The Lounge_ client

http://irc.lc/freenode/opensourcedesign

**Comment by jancborchardt** [comment]
Alternatively there’s also https://webchat.freenode.net/?channels=opensourcedesign – but of course the Lounge is a way nicer interface. :) (And I hope soon freenode webchat will be able to run it.)

https://github.com/thelounge/lounge
